cas/src/main/java/org/springframework/security/cas/web/CasAuthenticationEntryPoint.java

```diff
@@ -68,11 +69,18 @@ public void afterPropertiesSet() throws Exception {
         Assert.notNull(this.serviceProperties.getService(),"serviceProperties.getService() cannot be null.");
     }
 
+
+    public final void commence(final HttpServletRequest servletRequest, final HttpServletResponse response,
+                               final AuthenticationException authenticationException) throws IOException, ServletException {
+        // by default, don't force the renew parameter to true
+        commence(servletRequest, response, authenticationException, false);
+    }
+
     public final void commence(final HttpServletRequest servletRequest, final HttpServletResponse response,
-            final AuthenticationException authenticationException) throws IOException, ServletException {
+            final AuthenticationException authenticationException, boolean forceRenew) throws IOException, ServletException {
 
         final String urlEncodedService = createServiceUrl(servletRequest, response);
-        final String redirectUrl = createRedirectUrl(urlEncodedService);
+        final String redirectUrl = createRedirectUrl(urlEncodedService, forceRenew);
```

Comment:
it's not a good practice to pass boolean as parameter, just add another explicit method for this case